backend/aopp_test.go

```diff
@@ -183,4 +202,115 @@ func TestAOPPSuccess(t *testing.T) {
 			)
 		})
 	}
+
+	t.Run("error/wrong_version", func(t *testing.T) {
+		b := newBackend(t, testnetDisabled, regtestDisabled)
+		defer b.Close()
+		params := defaultParams()
+		params.Set("v", "1")
+		b.HandleURI("aopp:?" + params.Encode())
+		require.Equal(t, aoppStateError, b.AOPP().State)
+		require.Equal(t, errAOPPVersion, b.AOPP().ErrorCode)
+
+	})
+	t.Run("error/invalid_request", func(t *testing.T) {
+		b := newBackend(t, testnetDisabled, regtestDisabled)
+		defer b.Close()
+		params := defaultParams()
+		params.Del("callback")
+		b.HandleURI("aopp:?" + params.Encode())
+		require.Equal(t, aoppStateError, b.AOPP().State)
+		require.Equal(t, errAOPPInvalidRequest, b.AOPP().ErrorCode)
+	})
+	t.Run("error/invalid_request", func(t *testing.T) {
```

Comment:
not a good practice to give sub tests the same name: "invalid callback", "missing callback" for the one above this, "missing msg" for the one below, and so on.